akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -401,10 +404,10 @@ Links to reference documentation:
 
 ## Synchronous Testing
 
-The Test Kits for synchronous testing are completely different.
+Classic and typed have different Test Kits.
```

Comment:
This changes the meaning, I think the intention was to convey that the design of the synchronous testkit APIs are very different (TestActor vs BehaviorTestKit), this just reads as that they are two different test kits.

Reply:
How about: `Classic and typed have completely different Test Kits`?

Reply:
The complication is that this is just about the synchronous testing, the async testing is more similar between the two.

Reply:
or `Classic and typed have completely different Test Kits for synchronous testing`

Reply:
Sounds good to me :)

Reply:
I guess this is close to the original. What was missing there what the different _between x and y_ part

Reply:
Added my latest suggestion.